src/wikitextprocessor/core.py

```diff
@@ -1814,16 +1810,8 @@ def check_template_need_expand(
 
         return page.need_pre_expand
 
-    def read_by_title(
-        self, title: str, namespace_id: Optional[int] = None
-    ) -> Optional[str]:
-        """Reads the contents of the page.  Returns None if the page does
-        not exist."""
-        page = self.get_page_resolve_redirect(title, namespace_id)
-        return page.body if page is not None else None
```

Comment:
This old method get the name because it reads a huge cache file before we migrate to sqlite db. The new  `get_page_body()` now requires namespace id.